starter_project_backend_nest/src/user/user.model.ts

```diff
@@ -20,4 +21,9 @@ export const UserSchema = new mongoose.Schema({
     type: String,
     required: true,
   },
+  profileId: {
+    type: String,
```

Comment:
this type should be mongoose.Schema.ObjectId

Reply:
fixed